src/logic/wallets/pairing/module.ts

```diff
@@ -1,10 +1,13 @@
-import { IClientMeta } from '@walletconnect/types'
+import WalletConnectProvider from '@walletconnect/web3-provider'
+import { IClientMeta, IRPCMap } from '@walletconnect/types'
 import { WalletModule } from 'bnc-onboard/dist/src/interfaces'
 import UAParser from 'ua-parser-js'
 
-import { APP_VERSION, PUBLIC_URL } from 'src/utils/constants'
+import { APP_VERSION, INFURA_TOKEN, PUBLIC_URL, WC_BRIDGE } from 'src/utils/constants'
 import { ChainId } from 'src/config/chain'
-import { getWCWalletInterface, getWalletConnectProvider } from 'src/logic/wallets/walletConnect/utils'
```

Comment:
This logic was originally abstracted as it was used in the patched WC module as well as here.